test/e2e/network/service.go

```diff
@@ -3918,7 +3918,7 @@ func execAffinityTestForSessionAffinityTimeout(ctx context.Context, f *framework
 	framework.ExpectNoError(err)
 
 	// the service should be sticky until the timeout expires
-	framework.ExpectEqual(checkAffinity(ctx, cs, execPod, svcIP, servicePort, true), true)
+	gomega.Expect(checkAffinity(ctx, cs, execPod, svcIP, servicePort, true)).To(gomega.BeTrue())
```

Comment:
Can you also change this to not use `BeTrue`? See https://github.com/kubernetes/kubernetes/issues/105678.

Reply:
This isn't resolved?

Reply:
@pohly Sorry, it was resolved but I forgot remove this line.